src/views/ModelPlan/TaskList/ITSolutions/_components/OperationalNeedModalContext/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { createContext, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import { useMutation } from '@apollo/client';
+import { Button } from '@trussworks/react-uswds';
+
+import Modal from 'components/Modal';
+import PageHeading from 'components/PageHeading';
+import Alert from 'components/shared/Alert';
+import useMessage from 'hooks/useMessage';
+import {
+  UpdateCustomOperationalNeedById as MutationType,
+  UpdateCustomOperationalNeedByIdVariables
+} from 'queries/ITSolutions/types/UpdateCustomOperationalNeedById';
+import UpdateCustomOperationalNeedById from 'queries/ITSolutions/UpdateCustomOperationalNeedById';
+
+type ContextTypes = {
+  modelID: string;
+  id: string;
+  nameOther: string;
+};
+
+const OperationalNeedModalContext = createContext({
+  isModalOpen: false,
+  setIsModalOpen: (isModalOpen: boolean) => {},
+  operationalNeed: {
+    modelID: '',
+    id: '',
+    nameOther: ''
+  },
+  setOperationalNeed: (operationalNeed: ContextTypes) => {}
+});
```

Comment:
I'm not sure that `useContex`/`createContext` is the appropriate tool for this need.  I might not have the full context on its use.  We should pair on this to review this component and it's purpose.